scripts/VJMP-JP/c100200199.lua

```diff
@@ -0,0 +1,81 @@
+--深海艺术指导
+function c100200199.initial_effect(c)
+	aux.AddCodeList(c,78868119)

```

Comment:
should change to **aux.AddMaterialCodeList(c,78868119)** if https://github.com/Fluorohydride/ygopro-scripts/pull/1621 merged.